templates/jaasai/experts/omnivector.html

```diff
@@ -0,0 +1,171 @@
+      </div>
+      <ul>
+        <li class="p-list__item">4 Node SLURM starter bundle.</li>
+        <li class="p-list__item">Get up and running by deploying this bundle on your local machine, then deploy to your
+          favourite cloud when you are ready to take your solution live.</li>
+        <li class="p-list__item">SLURM packaging experts.</li>
+      </ul>
+      <a href="{{ external_urls.gui }}?dd=~omnivector/bundle/slurm-core" class="p-button--positive">Deploy SLURM Core</a>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3>
+        <a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='spark-jupyter-conda-extended') }}">
+          Big-data processing with CEPH and real-time monitoring.
+        </a>
+      </h3>
+      <p>Ominvector’s Spark Extended bundle provides a complete data processing solution at the push of a button.</p>
+      <div class="pictogram-bg u-hide--small">
+        <object data="{{ external_urls.charmstore }}~omnivector/bundle/spark-jupyter-conda-extended-0/diagram.svg"
+          type="image/svg+xml" style="min-height:280px" title="Spark Core overview"></object>
+      </div>
+      <ul>
+        <li class="p-list__item">Enhanced Spark Operations</li>
+        <li class="p-list__item">Integrated CEPH Storage</li>
+        <li class="p-list__item">Enterprise Grade Monitoring</li>
+      </ul>
+      <a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='spark-jupyter-conda-extended') }}" class="p-button--positive">
+        Deploy Spark Cluster with Monitoring
+      </a>
+    </div>
+    <div class="col-6">
+      <h3>
+        <a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='slurm-openfoam') }}">
+          Custom HPC Solutions</a>
+      </h3>
+      <p>Omnivector’s SLURM + OpenFOAM bundle exemplifies our experience automating operational solutions for HPC workloads.</p>
+      <div class="pictogram-bg">
+        <object data="{{ external_urls.charmstore }}~omnivector/bundle/slurm-openfoam/diagram.svg”" type="image/svg+xml"
+        style="min-height:280px" title="Omnivector’s SLURM + OpenFOAM bundle overview"></object>
+      </div>
+      <ul>
+        <li class="p-list__item">SLURM Core + OpenFOAM</li>
+        <li class="p-list__item">Easily integrate your OpenFOAM based solutions.</li>
+        <li class="p-list__item">Custom SLURM application packaging!</li>
+      </ul>
+      <a href="{{ external_urls.gui }}?dd=~omnivector/bundle/slurm-openfoam" class="p-button--positive">
+        Deploy SLURM OpenFOAM
+      </a>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip u-no-padding--bottom">
+  <div class="row">
+    <div class="col-4">
+      <h3 class="expert__title">About us</h3>
+      <p>Omnivector Solutions Limited is a Service Disabled Veteran Owned Small Business based in Portland, OR.</p>
```

Comment:
It looks like "Service-Disabled Veteran-Owned" (note hyphens) is how this is presented in other official documentations.

https://www.gsa.gov/small-business/find-and-pursue-government-contracts/seek-opportunities/setasides-special-interest-groups/for-servicedisabled-veteranowned-small-businesses